### ISSUE TITLE
key-format-style allowing incorrect format in locale file

### ISSUE THREAD

**Issue by alexcroox** [intlify/eslint-plugin-vue-i18n#152]
**Tell us about your environment**

"eslint": "^7.17.0",
"eslint-plugin-nuxt": "^2.0.0",
"@intlify/eslint-plugin-vue-i18n": "^0.9.0",
Node version: v10.15.3

**Please show your full configuration:**

```json5
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
  extends: ['@nuxtjs', 'prettier', 'prettier/vue', 'plugin:prettier/recommended', 'plugin:nuxt/recommended'],
  plugins: ['prettier', '@intlify/vue-i18n'],
  // add your custom rules here
  rules: {
    'nuxt/no-cjs-in-config': 'off',
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    // https://eslint.org/docs/rules/padding-line-between-statements
    'padding-line-between-statements': [
      'error',
      { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
      { blankLine: 'any', prev: ['const', 'let', 'var'], next: ['const', 'let', 'var'] },
      { blankLine: 'always', prev: ['case', 'default'], next: '*' },
      { blankLine: 'always', prev: 'directive', next: '*' },
      { blankLine: 'always', prev: 'function', next: '*' },
      { blankLine: 'always', prev: '*', next: 'function' },
      { blankLine: 'always', prev: 'if', next: '*' },
      { blankLine: 'always', prev: '*', next: 'if' },
      { blankLine: 'any', prev: 'directive', next: 'directive' }
    ],
    'lines-between-class-members': ['error', 'always'],
    'prefer-template': 'error',
    'vue/component-name-in-template-casing': ['error', 'PascalCase', { registeredComponentsOnly: true }],
    'vue/new-line-between-multi-line-property': [
      'error',
      {
        minLineOfMultilineProperty: 2
      }
    ],
    'vue/require-name-property': 'error',
    'vue/object-curly-spacing': ['error', 'always'],
    'vue/padding-line-between-blocks': ['error', 'always'],
    // https://eslint-plugin-vue-i18n.intlify.dev/rules/#recommended
    '@intlify/vue-i18n/no-html-messages': ['error'],
    '@intlify/vue-i18n/no-missing-keys': ['error'],
    '@intlify/vue-i18n/no-raw-text': 'off',
    '@intlify/vue-i18n/no-v-html': 'error',
    '@intlify/vue-i18n/key-format-style': ['error', 'snake_case'],
    '@intlify/vue-i18n/no-duplicate-keys-in-locale': [
      'error',
      {
        ignoreI18nBlock: false
      }
    ],
    '@intlify/vue-i18n/no-dynamic-keys': 'error',
    '@intlify/vue-i18n/no-unused-keys': [
      'off',
      {
        extensions: ['.js', '.vue']
      }
    ]
  },
  settings: {
    'vue-i18n': {
      localeDir: 'locales/*.json'
    }
  }
}

```

**What did you do?**

en-GB.json
```json5
{
  "WELCOME": "Welcome",
  "LOCALE_TEST_STRING": "This is a sentence in english",
  "thisShouldError": "if you see this eslint is not working as expected"
}
```

**What did you expect to happen?**
Eslint to error on the `thisShouldError` key because of `'@intlify/vue-i18n/key-format-style': ['error', 'snake_case']` is set to snake_case and the key is camelCase

**What actually happened?**
No error reported

**Comment by ota-meshi** [comment]
Thank you for this issue.
I don't know the cause of your problem. Would you please share a repository to reproduce the problem?

**Comment by gustavo-iha** [comment]
I believe the lib should be added in extensions:
```js
extends: ['plugin:@intlify/vue-i18n/recommended']
```
